### DISCUSSION TITLE
Who uses tfaction?

### DISCUSSION THREAD

**Discussion by suzuki-shunsuke** [suzuki-shunsuke/tfaction#1280] [Deprecated (General)]
Hi, I'd like to know which companies use tfaction.
And I'd like to add the section ~~`Which companies use tfaction?`~~ `Who uses tfaction?` to the document.
If your company uses tfaction, please tell us!
And also please tell us if it's okay to add your company to the list.

Thank you for your cooperation!

## List

- https://github.com/suzuki-shunsuke/tfaction#who-uses-tfaction

## How to contribute

Please send pull requests to [tfaction](https://github.com/suzuki-shunsuke/tfaction) to add your organization to the list.
If you don't want to send pull request, you can only post a comment to the discussion. Then we can send pull requests instead of you.

https://github.com/suzuki-shunsuke/tfaction/pulls?q=is%3Apr+label%3Awho-uses+is%3Aclosed

## Reference

- ArgoCD https://github.com/argoproj/argo-cd/blob/master/USERS.md

**Comment by rrreeeyyy** [comment]
Hello, I am @rrreeeyyy from Topotal Inc. (https://topotal.com/). Our company uses tfaction for managing our internal terraform environment. Additionally, as part of our technical support for SRE, we sometimes propose and implement tfaction for improving terraform CI/CD environments in our clients' setups. It would be an honor to have our company name listed if there is such a list. Thank you for the wonderful OSS!

**Reply by suzuki-shunsuke** [reply]
Thank you!
I'll add the list to the README of tfaction and the top page of the official document.

- https://github.com/suzuki-shunsuke/tfaction/pull/1327
- https://github.com/suzuki-shunsuke/tfaction-docs/pull/177

**Reply by suzuki-shunsuke** [reply]
Done.

**Reply by rrreeeyyy** [reply]
Thank you!

**Comment by kyontan** [comment]
As @suzuki-shunsuke wrote in the past in our product blog 😂 , we [StudySapuri / Quipper](https://github.com/quipper) team are continuously using tfaction in our terraform monorepo, and we have 500+ working directories (= states), and managing 11 external services (IaaS / SaaS) with tfaction.
Thanks for your perpetual support for tfaction and its ecosystem!

https://blog.studysapuri.jp/entry/2022/02/04/080000

**Reply by suzuki-shunsuke** [reply]
Thank you always! Is it okay to add the company name to the document (I expect it's difficult)?
If it's difficult, what should we add to the document?

**Reply by suzuki-shunsuke** [reply]
Hmm. `[StudySapuri / Quipper](https://github.com/quipper) team` may be one of options.

**Reply by kyontan** [reply]
I think that the fact that we are using `tfaction` at that moment is public (as written in blog), and we have no financial matters / conflicts of that, putting the name of product and/or company should not be a problem right?

**Reply by suzuki-shunsuke** [reply]
Yeah, you're right. Okay. I see. Maybe I was thinking too hard.
Thank you.

**Reply by suzuki-shunsuke** [reply]
Done.

- https://github.com/suzuki-shunsuke/tfaction/pull/1330
- https://github.com/suzuki-shunsuke/tfaction-docs/pull/179

**Comment by suzuki-shunsuke** [comment]
[Henry, Inc.](https://corp.henry-app.jp/)

Contributed to make [tfaction](https://github.com/suzuki-shunsuke/tfaction/releases/tag/v0.5.16) and [tfmigrate](https://github.com/minamijoyo/tfmigrate/releases/tag/v0.3.7) GoogleCloud-ready

- https://github.com/suzuki-shunsuke/tfaction/pull/1332
- https://github.com/suzuki-shunsuke/tfaction-docs/pull/181

**Comment by johnmanjiro13** [comment]
Hi, I'm @johnmanjiro13 from [Gunosy Inc.](https://gunosy.co.jp)
We use tfaction in almost all of our products to manage CI/CD of terraform.
I send a pull request (#1338) and I would be happy to add our company name to the list.
Thank you for your great OSS!

**Comment by suzuki-shunsuke** [comment]
tacoms introduced tfaction 🎉
https://zenn.dev/tacoms/articles/988e4c7efc8cf3
